lib/canvas_link_migrator/link_parser.rb

```diff
@@ -181,8 +181,8 @@ def resolved(new_url = nil)
 
     # returns a hash with resolution status and data to hold onto if unresolved
     def parse_url(url, node, attr)
-      if url =~ /wiki_page_migration_id=(.*)/
-        unresolved(:wiki_page, migration_id: $1)
+      if url =~ /wiki_page_migration_id=([^?]*)(\?.*)?/
```

Comment:
All right, I looked at the history of this, and while I don't love it, there aren't simpler ways.

It might be nice, though, if we did the same thing for the discussion_topics on the next level down, since we know it can be a problem. :)